docs/reference/ml/anomaly-detection/apis/put-datafeed.asciidoc

```diff
@@ -93,6 +103,20 @@ include::{docdir}/ml/ml-shared.asciidoc[tag=script-fields]
 (Optional, unsigned integer)
 include::{docdir}/ml/ml-shared.asciidoc[tag=scroll-size]
 
+
+[[ml-put-datafeed-time-related-settings]]
+===== Interaction of time-related settings
+
+Time-related settings have the following relationships:
+
+* When `query_delay` is present, queries run at `query_delay` after the end of 
```

Comment:
It might be misleading to say "When `query_delay` is present", because if it's not set then we give it a default value somewhere between 1 and 2 minutes.  So maybe change to just:

> Queries run at `query_delay` after the end of each frequency.